mycity/mycity/intents/speech_constants/snow_parking_intent.py

```diff
@@ -17,3 +17,4 @@
 NO_FEE = " There is no fee. "
 
 ERROR_SPEECH = "I need a valid address to find the closest parking"
+ERROR_INVALID_ADDRESS = "We are unable to use this address at this time"
```

Comment:
Lets add some more info:
"I can't seem to find this address. Please make sure to provide both the street number and street name."